### PR TITLE
Remove usage of multiple data sources and update default DB env var

### PR DESCRIPTION
**Pull request by 2color**
- Remove usage of multiple data sources
- Rename as per https://github.com/prisma/prisma2/issues/1481

### CODE REVIEW

File: schema/Readme.md [hunk 1]
```diff
@@ -86,23 +86,12 @@ your datasources with Lift and administer your data using Studio.
...
-
-datasource mgo {
-  provider = "mongodb"
-  url      = env.MONGO_URL
-}
-
-datasource mgo2 {
-  provider = "mongodb"
-  url      = env.MONGO2_URL
+  url      = env.DATABASE_URL
```
Comment (janpio) [inline schema/Readme.md]: Wasn't this changed to a function?

Comment (2color) [reply schema/Readme.md]: Yes. I just noticed. I'll update the PR accordingly

File: schema/Readme.md [hunk 2]
```diff
@@ -1104,7 +1092,7 @@ The schema can require certain environment expectations to be met. The purpose o
 ```groovy
 datasource pg {
   provider = "postgres"
-  url      = env("POSTGRES_URL")
+  url      = env("DATABASE_URL")
```
Comment (janpio) [inline schema/Readme.md]: Unrelated change to standardize to `DATABASE_URL`?

Comment (2color) [reply schema/Readme.md]: Yes. Can move to a different PR if you prefer.

Comment (janpio) [reply schema/Readme.md]: Adding to title is enough

**Comment by sorenbs** [comment]
Thank you 🙏
